.github/workflows/link-checker.yml

```diff
@@ -0,0 +1,43 @@
+# This Link Checker is run on all documentation files once per week.
+
+# references:
+# - https://github.com/lycheeverse/lychee-action
+# - https://github.com/lycheeverse/lychee
+
+name: Link Checker
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '30 8 * * 5'
```

Comment:
If we have triggers on push and pull_request, why do we need a schedule?
I think we should have a push trigger on master too, as that's where the docs are actually published from.

Reply:
Good question @sidharthv96.

This is how I used this on another repo, with lower activity. That way even if there are no pushes/prs in a give week, the links would still get checked once.

If the mermaid repo has a lot of activity anyways, which it looks like it does, you could remove the whole `schedule` block here.